nss-day-cohort-46/rare-server-nekorth-novices
# Jhf post ownership

# Description

## Type of change


# Testing Instructions
should be able to edit and delete posts that you made... only with client side buttons will not show if you did not make the post backend needs todo
## Postman URL

## Postman Type (Get, delete, put, etc.)

## Server Side Instructions

## Client Side Instructions
use branch jhf-edit-delete-buttons
# Checklist:
